libraries/AP_ESC_Telem/AP_ESC_Telem.cpp

```diff
@@ -527,7 +527,7 @@ void AP_ESC_Telem::update()
                 const struct log_Esc pkt{
                     LOG_PACKET_HEADER_INIT(uint8_t(LOG_ESC_MSG)),
                     time_us     : now_us64,
-                    instance    : i,
+                    instance    : _i,  //makes logged instance agree to output that ESC is attached
```

Comment:
probably no need for the intermediate "_i" variable.  This line could be changed to: "instance : i+1" instead.

There's a comment a few lines up that says "id starts from 0" which should be updated.

Reply:
tried that first and it failed to compile